test/pools/disjoint_pool.cpp

```diff
@@ -57,10 +57,17 @@ TEST_F(test, freeErrorPropagation) {
     static constexpr size_t size = 1024;
     void *ptr = umfPoolMalloc(pool, size);
 
+    // Check for simulated free error propagation
```

Comment:
I believe this fix is already merged to main. Please rebase

Reply:
Done.

Reply:
Please rebase again, I can see that the fix for memoryPoolAPI.cpp has been merged to main.
Also, I guess you want to update Igor's commit as well...?